bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,18 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
```

Comment:
No, thats fine. Thanks for explaining. As far as I'm concerned you can leave it.